src/main/kotlin/no/nav/familie/ef/sak/blankett/BlankettService.kt

```diff
@@ -24,7 +24,9 @@ class BlankettService(private val tilgangService: TilgangService,
         tilgangService.validerTilgangTilPersonMedBarn(personIdent)
         val søknad = journalføringService.hentSøknadFraJournalpostForOvergangsstønad(journalpostId)
         val fagsak = fagsakService.hentEllerOpprettFagsak(personIdent, Stønadstype.OVERGANGSSTØNAD)
-        return behandlingService.opprettBehandling(BehandlingType.BLANKETT, fagsak.id, søknad, journalpost)
+        val opprettBehandling = behandlingService.opprettBehandling(BehandlingType.BLANKETT, fagsak.id, søknad, journalpost)
+        lagreTomBlankett(opprettBehandling.id)
```

Comment:
Hvorfor trengs denne? Hvorfor ikke bruke lagBlankettPdf i stedet?

Reply:
Det er ikke noe å lage pdf med ennå. 
Dette skjer når en saksbehandler "plukker oppgave". 
Når vi senere genererer pdf (etter vilkårsvurdering) vil den genererte pdf'en legges inn i "blankett". 
(Dette får å slippe kode som må sjekke om vi skal gjøre en "insert or update" i generer pdf. 
 
Blankett her burde kanskje endres til lagre tomBlankettBehandling?

Reply:
Skal ikke blanketten opprettes av en task når man godkjent vedtaket, slik att det burde kun være en insert?

Reply:
Tror vi har tenkt at saksbehandler 1 skal se blanketten hen sender til totrinn: 
Forsøker meg på tenkt (happyday) flyt her: 
Kanskje det kan bli slik? 
1. Vi oppretter blankettbehandling (bahandling type blankett)
2. Saksbehandler 1 gjør vilkårsvurdering mm.
3. Vi genererer blankett for saksbehandler 1 (viser fram) 
4. Sender til totrinn
5. Saksb. 2 ser på blankett (evt. vilkårsvurderinger mm) 
6. Saksb. 2 godkjenner. 
7. Blankett arkiveres 
8. Ferdigstill